sdk/zklogin/src/checks.ts

```diff
@@ -69,8 +69,8 @@ export function lengthChecks(
 	/// Check that nonce extended nonce length is as expected.
 	const nonce_claim_details = processor.process('nonce');
 	const nonce_value_len = nonce_claim_details.offsets.value_length;
-	if (nonce_value_len !== 27) {
-		throw new Error(`Nonce value length is not 27`);
+	if (nonce_value_len !== 29) {
```

Comment:
it might be nice to throw this in a constant, i.e.`NONCE_LENGTH`, to clarify the intent & not have to change `29` in both places when making future updates 😄

Reply:
Ah! Good suggestion. Changed it, thanks!